include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1850,6 +1850,20 @@ int bt_audio_stream_config(struct bt_conn *conn,
 			   struct bt_audio_ep *ep,
 			   struct bt_codec *codec);
 
+/** @brief Configure Audio Stream
+ *
+ *  This procedure is used by a server to configure a new stream using the
+ *  ASE ID and codec configuration.
+ *
+ *  @param conn Connection object
+ *  @param ase_id ID of Audio Stream Endpoint being configured
+ *  @param codec Codec configuration
+ *
+ *  @return 0 in case of success or negative value in case of error.
+ */
+int bt_audio_stream_config_srv(struct bt_conn *conn, int ase_id,
+				struct bt_codec *codec);
```

Comment:
In the ASCS/SR/ACP/BV-07-C test case the ase_id is given in MMI message, so it can be parsed by AutoPTS and sent to the IUT. But I don't know about other applications.

Reply:
I made a refactor and included the `ase_id` parameter in the definition of `bt_audio_stream_config`. Is that ok? Other option I've considered was creating in public api a getter like
`bt_audio_ep *ascs_ep_get(struct bt_conn *conn, uint8_t ase_id);`
and then just pass the retrieved *ep to the function `bt_audio_stream_config`.